spec.go

```diff
@@ -409,6 +420,23 @@ func setupUserNamespace(spec *specs.LinuxSpec, config *configs.Config) error {
 	return nil
 }
 
+func setupRlimits(spec *specs.LinuxSpec, config *configs.Config) error {
+	if len(spec.Linux.Rlimits) == 0 {
+		return nil
+	}
+	create := func(rlimit specs.Rlimit) configs.Rlimit {
+		return configs.Rlimit{
+			Type: int(rlimit.Type),
+			Hard: uint64(rlimit.Hard),
+			Soft: uint64(rlimit.Soft),
+		}
+	}
+	for _, rlimit := range spec.Linux.Rlimits {
+		config.Rlimits = append(config.Rlimits, create(rlimit))
+	}
+	return nil
```

Comment:
it always return nil (no error), I think we should remove the return type (error) from this function.